src/components/textToSpeech/js/TextToSpeech.js

```diff
@@ -145,7 +145,8 @@ fluid.defaults("fluid.textToSpeech", {
         utteranceOpts: {
             // text: "", // text to synthesize. Avoid using, it will be overwritten by text passed in directly to a queueSpeech
             // lang: "", // the language of the synthesized text
-            // voice: {} // a WebSpeechSynthesis object; if not set, will use the default one provided by the browser
+            // voice: {} // a WebSpeechSynthesis object; it must be set in safari. In other browsers, if not set,
+            //              will use the default one provided by the browser
```

Comment:
I think you should probably [file a bug](https://bugs.webkit.org) against Webkit and indicate the issue in the comments here. According to [Bug 237614](https://bugs.webkit.org/show_bug.cgi?id=237614) they should have updated to the latest version of the Web Speech API. Which indicates that the [voice attribute](https://wicg.github.io/speech-api/#dom-speechsynthesisutterance-voice) is nullable and states 

> If this attribute is unset or null at the time of the [speak()](https://wicg.github.io/speech-api/#dom-speechsynthesis-speak) method call, then the user agent must use a user agent default voice.

I updated my computer this morning and got the latest Safari 15.6 and also tried the [Safari Technology Preview](https://developer.apple.com/safari/resources/) with the build site, and neither work. So unfortunately this is still an issue.